pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -309,7 +309,13 @@ func publishAppInstanceStatus(ctx *zedmanagerContext,
 	key := status.Key()
 	log.Debugf("publishAppInstanceStatus(%s)", key)
 	pub := ctx.pubAppInstanceStatus
-	pub.Publish(key, *status)
+	// zedmanager AppInstanceStatus include UnderlayNetworkConfig twice, remove one
```

Comment:
I don't understand what twice means. 
And the large thing is not the ACLs field but the ACLRules field AFAIK.

Reply:
Does anything outside of zedrouter use the ACLs in AppNetworkStatus? If not we can also move that to the local struct.

Reply:
> I don't understand what twice means.
> And the large thing is not the ACLs field but the ACLRules field AFAIK.

the UnderlayNetworkStatus include the struct of UnderlayNetworkConfig{} which has the list of ACEs, and this zedmanager AppInstanceStatus includes both []UnderlayNetworkStatus, and []UnderlayNetworkConfig. Agreed that the ACLs fields (basically 250 bytes per ACL) is much smaller than the ACLRules. I can certainly remove this optimization here.

Reply:
> Does anything outside of zedrouter use the ACLs in AppNetworkStatus? If not we can also move that to the local struct.

I saw something like domainmgr when format domainConfig it uses the AppNetworkStatus at least for the AppNum there.

Reply:
Sure the AppNum and MacAddrs are used. But the ACLs?

Reply:
I see, I had added this to AppInstanceStatus causing the dual inclusion:
	UnderlayNetworks    []UnderlayNetworkStatus
	// Copies of config to determine diffs
	UnderlayNetworkList []UnderlayNetworkConfig

I wonder if we can redo that logic to not include all of UnderlayNetworkConfig as part of the determination of diffs. 
If you clear the field before publish the diff determination might fail.

Reply:
Note that we do Get() and GetAll() of what we published, and if you clear the field before publishing you won't get it back from those. So I think we need to put those copies which are used to look for diffs in a separate map.

Reply:
as we discussed offline. this PR stays to remove the ACLRules from the UnderLayNetworkStatus. you have some other changes on the way to further optimize.